plugins/bil/bil_main.ml

```diff
@@ -0,0 +1,77 @@
+open Core_kernel
+open Bap.Std
+open Regular.Std
+
+include Self ()
+
+(* variable is considred dead if it's a virtual and equal to const and
+   assigned only once. Assuming const propagation was ran before *)
```

Comment:
why do we remove only constant assignments? What's wrong with removing a bogus `v = RAX`?